src/components/SearchResults.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "../styles/search-results.css";
+
+const SearchResults = () => {
+  return (
+    <>
+      <p>Search Results</p>
+      <img
+        className="card-image"
+        src="https://images.unsplash.com/photo-1522030299830-16b8d3d049fe?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80"
+        alt="space-image"
```

Comment:
You should avoid using the word `image` in an alt attribute for an image. This is because a screen-reader will already identify this element as an image.